docs/src/main/tut/docs/library-integrations.md

```diff
@@ -11,6 +11,7 @@ The core of PureConfig eschews unnecessary dependencies. Separate modules exist
 
 - [`pureconfig-akka`](https://github.com/pureconfig/pureconfig/tree/master/modules/akka) provides converters for [Akka](http://akka.io/) data types;
 - [`pureconfig-cats`](https://github.com/pureconfig/pureconfig/tree/master/modules/cats) provides converters for [Cats](http://typelevel.org/cats/) data structures and Cats typeclass instances;
+- [`pureconfig-cats-effect`](https://github.com/pureconfig/pureconfig/tree/master/modules/cats-effect) uses [cats-effect](https://github.com/typelevel/cats-effect) to load and save configurations in a side-effect free manner.
```

Comment:
Nitpick: can you terminate the description with a `;` in order to keep the style of the remaining modules? :)